pkg/crc/preflight/preflight_checks_tray_windows.go

```diff
@@ -204,3 +201,34 @@ func stopTray() error {
 	}
 	return nil
 }
+
+func checkUserHasServiceLogonEnabled() error {
+	username := getCurrentUsernmae()
+	enabled, err := secpol.UserAllowedToLogonAsService(username)
+	if enabled {
+		return nil
+	}
+	return fmt.Errorf("Failed to check if user is allowed to log on as service: %v", err)
+}
+
+func fixUserHasServiceLogonEnabled() error {
+	// get the username
+	username := getCurrentUsernmae()
+
+	return secpol.AllowUserToLogonAsService(username)
+}
+
+func getCurrentUsernmae() string {
+	user, err := user.Current()
+	if err != nil {
+		logging.Debugf("Unable to get current user's name: %v", err)
+		return ""
+	}
+	return strings.TrimSpace(strings.Split(user.Username, "\\")[1])
+}
+
+func disableUserServiceLogon() error {
```

Comment:
this might not have been set by us... so, just like other fixes we might want to record what we have modified to make CRC work (and undo this on a cleanup... only the failed issues+fixes that were applied)

Reply:
Yeah, no way to know if we made those changes, yes future improvement!